### PR TITLE
Clean up feature gates ServiceLoadBalancerClass and ServiceLBNodePortControl 

### PR DESCRIPTION
**Pull request by andrewsykim**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Both ServiceLoadBalancerClass and ServiceLBNodePortControl went GA in v1.24. 

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
The feature gates ServiceLoadBalancerClass and ServiceLBNodePortControl have been removed. These feature gates were enabled (and locked) since v1.24. 
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/features/kube_features.go [hunk 1]
```diff
@@ -761,24 +761,6 @@ const (
 	// Subdivide the ClusterIP range for dynamic and static IP allocation.
 	ServiceIPStaticSubrange featuregate.Feature = "ServiceIPStaticSubrange"
 
-	// owner: @andrewsykim @uablrek
```
Comment (andrewsykim) [inline pkg/features/kube_features.go:764]: FYI @uablrek

File: pkg/features/kube_features.go [hunk 2]
```diff
@@ -761,24 +761,6 @@ const (
...
-	// kep: http://kep.k8s.io/1864
-	// alpha: v1.20
-	// beta: v1.22
-	// ga: v1.24
-	//
-	// Allows control if NodePorts shall be created for services with "type: LoadBalancer" by defining the spec.AllocateLoadBalancerNodePorts field (bool)
-	ServiceLBNodePortControl featuregate.Feature = "ServiceLBNodePortControl"
-
-	// owner: @andrewsykim @XudongLiuHarold
```
Comment (andrewsykim) [inline pkg/features/kube_features.go:773]: FYI @XudongLiuHarold

File: pkg/features/kube_features.go (hunk 1 above)
Comment (uablrek) [reply pkg/features/kube_features.go:764]: @andrewsykim Thanks, and welcome back! :smile:

File: pkg/features/kube_features.go (hunk 2 above)
Comment (XudongLiuHarold) [reply pkg/features/kube_features.go:773]: Thanks @andrewsykim , welcome back!

**Comment by thockin** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**